product_docs/docs/edbcloud/beta/using_cluster/02_connect_to_cluster.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: "Connecting to Your Cluster"
+---
+You can connect to your cluster using [`psql`](http://postgresguide.com/utilities/psql.html), the terminal-based client for Postgres, or another client.
+
+To connect to your cluster using `psql`:  
+1. Go to the **Clusters** page.
+2. Select the name of your cluster. 
+3. On the **Overview** tab, select the copy icon to the right of the **Quick Connect** field to copy the command for connecting to your cluster using `psql`. 
+4. Paste the command in your `psql` terminal. 
+
+To connect to your cluster using a client other than `psql`:
+1. Go to the **Clusters** page.
+2. Select the name of your cluster. 
+3. Select the **Connect** tab. You can review and copy all the relevant information you need from this screen except for the user password. Please consult the client driver documentation for the connection string format the driver uses.  
```

Comment:
```suggestion
3. Select the **Connect** tab. You can review and copy all the relevant information you need from this screen except for the `edb_admin` user password. Please consult the client driver documentation for the connection string format the driver uses.  
```

Reply:
For the future, it might be nice to include an example or two here - maybe JS and Java?